ee/disclaim/run_disclaimed_darwin_test.go

```diff
@@ -0,0 +1,29 @@
+//go:build darwin
+// +build darwin
+
+package disclaim
+
+import (
+	"testing"
+
+	"github.com/kolide/launcher/pkg/log/multislogger"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRunDisclaimedDoesNotRunArbitraryCommands(t *testing.T) {
```

Comment:
Love the negative tests. Should we get a positive one? (Even if we can't really test the _disclaim_ part, we can probably test the exec and output passing.

(No need to block PR for it)